raiden-dapp/src/views/TransferRoute.vue

```diff
@@ -40,15 +43,47 @@ import { Token, TokenModel } from '@/model/types';
     NoChannelsDialog,
   },
   computed: {
+    ...mapState(['latestStateBackupReminder']),
     ...mapGetters(['tokens', 'channelWithBiggestCapacity', 'openChannels']),
   },
 })
 export default class TransferRoute extends Vue {
+  latestStateBackupReminder!: number;
   tokens!: TokenModel[];
+  currentTime: number = new Date().getTime();
+  oneDay: number = new Date().setHours(24);
+  stateBackupReminder = {
+    icon: this.$t('notifications.backup-state.icon') as string,
+    title: this.$t('notifications.backup-state.title') as string,
+    link: this.$t('notifications.backup-state.link') as string,
+    dappRoute: this.$t('notifications.backup-state.dapp-route') as string,
+    description: this.$t('notifications.backup-state.description') as string,
+    importance: NotificationImportance.HIGH,
+    context: NotificationContext.WARNING,
+  } as NotificationPayload;
   channelWithBiggestCapacity!: (
     tokenAddress: string
   ) => RaidenChannel | undefined;
 
+  mounted() {
+    if (this.latestStateBackupReminder === 0) {
+      this.pushStateBackupNotification();
+    } else if (
+      this.currentTime >
+      this.latestStateBackupReminder + this.oneDay
+    ) {
+      this.pushStateBackupNotification();
+    }
```

Comment:
Both condition execution blocks do 100% the same. So please merge the both boolean expression with an OR. If you like to you can store each expression into a boolean variable with a expressive name. In the end it could then read if `if (neverNotified || lastNotificationOlderThanOneDay) {...}`.

Reply:
Genau, good point. 👍